README.md

```diff
@@ -1,2 +1,195 @@
-# dogecoin.com
+# Dogecoin.com
 Temporary repo for dogecoin.com redesign
+
+## How to get up and running:
+I'm working off of a redesign branch and reconfigured gh-pages.yml to exclusively build and deploy this branch name. After you clone in order to make this work for github pages please checkout a new branch called redesign in your respective fork by doing:
+
+`git checkout -b redesign main`
+
+I've added bootstrap to help easily facilitate content alignment so you'll need to have Node.js installed: https://nodejs.org/en/
+
+After cloning the repo to your local run:
+
+`npm i`
+
+This will add bootstrap into your node_modules folder. bootstrap.min.js and bootstrap.min.css are minified on each hugo build so there's nothing more to do with regards to config. After that you can add any bootstrap class to frontend html or you can add custom css/scss to any of the style sheets in /themes/assets/scss or /themes/assets/css/
+
```

Comment:
I think this README is missing the command to run the dev server

Reply:
ah yes it is. you can still use hugo build or hugo server and or you can use npm hugo:build npm hugo:serve if you'd like. the npm commands aren't that important it's just to get bootstrap installed and then hugo takes over from there.